pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -145,9 +150,6 @@ func (r *ReconcileWindowsMachineConfig) Reconcile(request reconcile.Request) (re
 			return reconcile.Result{}, fmt.Errorf("error instantiating cloud provider: %v", err)
 		}
 	}
-	if r.k8sclientset == nil {
```

Comment:
/Why are you removing this check?

Reply:
I moved it to the reconciler creation. Is it possible for it to be removed out from under us?

Reply:
It shouldnt be, even if the clientset is invalidated the pointer in the ReconcileWindowsMachineConfig struct wont change